packages/Python/lldbsuite/test/lang/swift/expression/simple/TestSimpleExpressions.py

```diff
@@ -31,14 +31,31 @@ def test_simple_swift_expressions(self):
         self.build()
         self.do_test()
 
+    @decorators.swiftTest
+    def test_frame_var_accelerator(self):
+        self.build()
+        self.do_locals_test()
+
     def setUp(self):
         TestBase.setUp(self)
         self.main_source = "main.swift"
         self.main_source_spec = lldb.SBFileSpec(self.main_source)
 
+    def do_locals_test(self):
+        """Test simple swift expressions"""
+        (target, process, self.thread, breakpoint) = lldbutil.run_to_source_breakpoint(self, 
+                                                                                        "Set breakpoint here", 
+                                                                                        self.main_source_spec)
+        self.frame = self.thread.frames[0]
+        self.assertTrue(self.frame, "Frame 0 is valid.")
+        self.expect("expression --allow-jit 0 -- same_five", substrs=["Bool", "true"])
```

Comment:
Could you please add a comment explaining why --allow-jit is used here? It's not obvious to me and perhaps others can learn something, too :-)